parse.dylan

```diff
@@ -522,6 +522,10 @@ define inline function parse-simple-group
                 let re = parse-regex(str, info);
                 if (lookahead(str) == ')')
                   consume(str)
+		else
+		  parse-error(str.parse-string,
+			      "Unterminated group at index %s.",
+			      str.parse-index);
```

Comment:
There's nothing on this in the style guide https://opendylan.org/documentation/style-guide/index.html but I like to use %d for integer formatting rather than %s. I don't know why. :-)  Up to you.

Please remove Tab chars because they can mess up indentation, for example here in github reviews..